src/lib/env.js

```diff
@@ -40,21 +38,12 @@ export class Env {
     return parser.parse.call(parser, emit, data);
   }
 
-  _createEntries(lang, ast) {
-    const entries = Object.create(null);
-    const create = lang.src === 'qps' ?
-      createPseudoEntry : createEntry;
-
-    for (let i = 0, node; node = ast[i]; i++) {
-      const id = node.$i;
-      if (id in entries) {
-        this.emit('duplicateerror', new L10nError(
-         'Duplicate string "' + id + '" found in ' + lang.code, id, lang));
```

Comment:
Absolutely, it's a feature of the views system: the order is significant.  For BTO, I think it's okay to be a bit more lax with dupes in l20n.js, but detect all of them in (legacy) l10n.js.